build.sbt

```diff
@@ -71,11 +71,15 @@ ThisBuild / credentials += Credentials("GnuPG Key ID", "gpg", "F68F0EADDB81EF533
 
 ThisBuild / scalacOptions ++= {
   CrossVersion.partialVersion(scalaVersion.value) match {
-    case Some((2, 13)) => Nil
+    case Some((2, 13)) => Seq(
+      "-deprecation"
+    , "-Wconf:cat=deprecation&msg=Auto-application:silent"
+    )
     case _ => Seq(
       "-source", "future-migration", "-deprecation"
     , "release", "11"
     , "-Yexplicit-nulls"
+    // , "-Ycheck:macros"
```

Comment:
@YuriyMazepin Have you seen this option before? I am not quite sure what it doing and why it gives errors like:
```
[error] 126 |  given MessageCodec[Message] = caseCodecAuto
[error]     |                                ^^^^^^^^^^^^^
[error]     |Exception occurred while executing macro expansion.
[error]     |java.lang.AssertionError: assertion failed: Tree had an unexpected owner for val p
[error]     |Expected: val msg1Prepare (proto.models$._$_$msg1Prepare)
[error]     |But was: val macro (proto.models$._$macro)
```

Reply:
According to source code this option "Check some invariants of macro generated code while expanding macros". As I understand it is useful for debugging complex macros. The reason for the error above is that the two value definitions had different context owners.